docs/design/overview.md

```diff
@@ -14,9 +14,9 @@ To get started on your UI, follow the below steps.
 
 ### Preferred: React+MUI, using our theme
 
-Docker Desktop's UI is written in React and [MUI](https://mui.com/) (a.k.a. Material UI). This is the only officially supported framework for building extensions, and the one that our `init` command automatically configures for you. Using it brings significant benefits to authors:
+Docker Desktop's UI is written in React and [MUI](https://mui.com/) (using Material UI to specific). This is the only officially supported framework for building extensions, and the one that our `init` command automatically configures for you. Using it brings significant benefits to authors:
```

Comment:
@oliviertassinari Could you explain why this change? Thanks!

Reply:
MUI is a company, it's not a product